sdk/resourcehealth/Azure.ResourceManager.ResourceHealth/api/Azure.ResourceManager.ResourceHealth.netstandard2.0.cs

```diff
@@ -76,6 +80,11 @@ public partial class ResourceHealthEventImpactedResourceData : Azure.ResourceMan
     {
         internal ResourceHealthEventImpactedResourceData() { }
         public System.Collections.Generic.IReadOnlyList<Azure.ResourceManager.ResourceHealth.Models.ResourceHealthKeyValueItem> Info { get { throw null; } }
+        public string MaintenanceEndTime { get { throw null; } }
```

Comment:
What the format of the value of this property? If this is [DataTimeOffset](https://learn.microsoft.com/en-us/dotnet/api/system.datetimeoffset?view=net-7.0), then please change the format and rename it to `MaintenanceEndOn`